### PR TITLE
Add ext_oneapi_unified_runtime SYCL backend test

### PR DESCRIPTION
**Pull request by smaslov-intel**
E2E test for https://github.com/intel/llvm/pull/7765
Signed-off-by: Sergey V Maslov <sergey.v.maslov@intel.com>

### CODE REVIEW

**Review by aelovikov-intel — APPROVED** [review]

File: SYCL/Plugin/sycl-ls-unified-runtime.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// REQUIRES: level_zero
+
```
Comment (smaslov-intel) [inline SYCL/Plugin/sycl-ls-unified-runtime.cpp]:
```suggestion
//
```

**Comment by againull** [comment]
Only failure SYCL :: SubGroupMask/Basic.cpp is known and tracked.

**Pull request merged**